playbooks/aws/openshift-cluster/build_ami.yml

```diff
@@ -39,6 +39,7 @@
 - import_playbook: ../../openshift-node/private/image_prep.yml
   vars:
     openshift_node_reboot_instance_before_cleanup: true
+    openshift_aws_ami_set_gquota_on_slash: false
```

Comment:
@sdodson Also note ... we condition the gquota tasks here.  We advertise the use of the Red Hat Cloud Access AMI's so we default to True.  A customer can flip this to false and skip the gquota tasks.